styles.css

```diff
@@ -0,0 +1,3 @@
+header {
```

Comment:
Please add a class to the `<h1>` element. This is part of the requirements
<img width="1041" alt="Screenshot 2022-09-21 at 14 49 01" src="https://user-images.githubusercontent.com/48546992/191758798-04622654-d48b-4544-8f6e-352acd30b12e.png">